_docs/02.5-deploy_app.markdown

```diff
@@ -33,7 +33,7 @@ Password:
 
 You should see a terminal message `Logged in as your-example-email@walmartlabs.com`. Now, let's deploy!
 
-We need to specify the [version of node](https://devcenter.heroku.com/articles/node-best-practices) you are using on your machine into our `package.json`. Add the code below the `author` section and before the `contributors` section of your `package.json`:
+You'll need to specify the [version of node](https://devcenter.heroku.com/articles/node-best-practices) you are using on your machine into our `package.json`. Add the following json below the `author` section and before the `contributors` section in your `package.json`:
```

Comment:
Add the following json below the `author` section and before the `contributors` section in your `package.json`

This is not a requirement, you can add it anywhere inside `package.json`